src/sentry/models/project.py

```diff
@@ -279,27 +281,35 @@ def get_mail_alert_subscribers(self):
         return list(User.objects.filter(id__in=user_ids))
 
     def is_user_subscribed_to_mail_alerts(self, user):
-        from sentry.models import UserOption
+        """
+        Check the user's "Issue Alert" Notification Settings, if they are
+        "default", then check if they have opted to subscribe by default.
+        :param user: User
+        :return: boolean
+        """
+        from sentry.models import NotificationSetting, UserOption
 
-        is_enabled = UserOption.objects.get_value(user, "mail:alert", project=self)
-        if is_enabled is None:
-            is_enabled = UserOption.objects.get_value(user, "subscribe_by_default", "1") == "1"
-        else:
-            is_enabled = bool(is_enabled)
-        return is_enabled
+        value = NotificationSetting.objects.get_settings(
```

Comment:
I thought this PR was just for writing, but this appears to be reading, right? Same as the other change in this file.

Reply:
You're right, let me move these to another PR.